core/server/data/meta/index.js

```diff
@@ -66,7 +66,7 @@ function getMetaData(data, root) {
     if (settingsCache.get('logo')) {
         metaData.blog.logo.url = utils.url.urlFor('image', {image: settingsCache.get('logo')}, true);
     } else {
-        metaData.blog.logo.url = utils.url.urlJoin(utils.url.urlFor('admin'), 'img/ghosticon.jpg');
+        metaData.blog.logo.url = utils.url.urlJoin(utils.url.urlFor('admin'), 'assets/img/ghosticon.jpg');
```

Comment:
We're talking about this line https://github.com/TryGhost/Ghost-Admin/blob/master/app/index.html#L22

Currently Ghost-Admin assumes the blog is available on the same domain which isn't always true. @AileenCGN also pointed out that it's incorrect in linking to `favicon.ico` as that will use an uploaded favicon if there is one but the admin should always be using the default favicon - I wasn't aware of this requirement at the time of making the change in index.html.

In light of both of those, having a default admin favicon in Ghost-Admin would be preferable - there's no assumption that the blog is available on the same domain and we'll always serve the default icon.

Sorry if the discussion veered too far from the original issue here.